### ISSUE TITLE
Finalize MVP Levels

### ISSUE THREAD

**Issue by zicklag** [fishfolk/jumpy#581]
We need at least 4, rock-solid level designs for the MVP.

We want to do this ASAP.

There are also 2 or 3 other maps from the experimental maps list where the semi-solid platforms are working fine and we just need to place items on the map. It'd be good to migrate those to the default maps once we nail them down, too.

**Comment by zicklag** [comment]
Waiting for at least mines from #554 before we can do this, after that, we want to do this ASAP.

There are also 2 or 3 other maps from the experimental maps list where the semi-solid platforms are working fine and we just need to place items on the map. It'd be good to migrate those to the default maps once we nail them down, too.

**Comment by ThousandthStar** [comment]
Should maps like the first one prevent the player from jumping off it? ATM it is possible (event though a bit tricky) to fall off the map.

**Comment by zicklag** [comment]
With https://github.com/fishfolk/jumpy/pull/617 being merged, we now have all of our MVP items re-implemented! 🎉 That means we can move forward with this.

**Comment by zicklag** [comment]
I'm going to be trying to get the editor ( #620 ) working to make this easier.

**Comment by zicklag** [comment]
Closing in favor of #697.
